low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -59,9 +59,13 @@ class RAIDIntegritySensor(SensorThread, InternalMsgQ):
     RACK_ID = "rack_id"
     POLLING_INTERVAL = "polling_interval"
     TIMESTAMP_FILE_PATH_KEY = "timestamp_file_path"
+    FAULT_ACCEPTED_TIME = "fault_accepted_time"
 
-    # check once a week (below time is in seconds), the integrity of raid data
-    DEFAULT_POLLING_INTERVAL = "604800"
+    # check once in two weeks (below time is in seconds), the integrity of raid data
```

Comment:
Update comment
\# Scan for RAID integrity error every 2 weeks (1209600 seconds)

Reply:
comment not addressed

Reply:
addressed.